components/TextFields/src/MDCMultilineTextField.m

```diff
@@ -685,22 +707,17 @@ - (void)setTextInsetsMode:(MDCTextInputTextInsetsMode)textInsetsMode {
 }
 
 - (UITextView *)textView {
-  if (!_textView) {
-    _textView = [[UITextView alloc] initWithFrame:CGRectZero];
-    [self setupTextView];
-    [self setupUnderlineConstraints];
-  }
+  [self textViewSafetyCheck];
```

Comment:
The previous implementation lazy-init'd out textView.
Do we no longer need to do this?  (textViewSafetyCheck does not lazy-init)

Reply:
Yes it does.
`  if (!_textView) {
    _textView = [[MDCIntrinsicHeightTextView alloc] initWithFrame:CGRectZero];
`